### ISSUE TITLE
First character missing from values of global bool constants ('rue', 'alse', 'ull')

### ISSUE THREAD

**Issue by radpuj** [phpDocumentor/phpDocumentor#2866]
## Steps to reproduce the problem
1. Create a simple PHP file:
```
<?php
  /**
   * This is description.
   *
   * @var boolean
   */
  const MY_TEST = false;
?>
```
2. Generate API docs with phpDocumentor.
3. Open the docs in the browser. They will contain **MY_TEST = alse** instead of **false** in multiple places. See [this screenshot](https://i.imgur.com/CeuPbET.png).
4. The same thing happens for values **true** and **null**.

## Your environment
phpDocumentor v3.0.0 with PHP 7.4.15 under Windows, default settings.

**Comment by cherrador** [comment]
I can verify that this is happening also on phpDocumentor v3.1.0

**Comment by mvriel** [comment]
As with most things that seem simple; this one was quite interesting to track down and I had to revert part of a fix of #2421 in order  to make sure no unwanted behaviour was exhibited

**Comment by mvriel** [comment]
The issue has been fixed on master and is included in the latest build of our docker image, or included in the next tagged release. If you would like access to the fix rather sooner than later you could consider using our nightly build (see Github Actions for the nightly build artifact)
